SocketRocket/SocketRocket/Internal/Delegate/ARTSRDelegateController.h

```diff
@@ -9,7 +9,7 @@
 
 #import <Foundation/Foundation.h>
 
-#import <SocketRocketAblyFork/ARTSRWebSocket.h>
```

Comment:
@maratal I removed `SocketRocketAblyFork` alias and changed import here

Reply:
The main issue is not removing the alias, but actually moving all the code in SocketRocket into Ably. There is no clear reason to me.

Reply:
They were separated into 2 modules again 👍